src/k8s/pkg/k8sd/setup/util_kubeconfig.go

```diff
@@ -69,7 +69,7 @@ func SetupControlPlaneKubeconfigs(kubeConfigDir string, securePort int, pki pki.
 		{file: "scheduler.conf", crt: pki.KubeSchedulerClientCert, key: pki.KubeSchedulerClientKey},
 		{file: "kubelet.conf", crt: pki.KubeletClientCert, key: pki.KubeletClientKey},
 	} {
-		if err := Kubeconfig(filepath.Join(kubeConfigDir, kubeconfig.file), fmt.Sprintf("127.0.0.1:%d", securePort), pki.CACert, kubeconfig.crt, kubeconfig.key); err != nil {
+		if err := Kubeconfig(filepath.Join(kubeConfigDir, kubeconfig.file), fmt.Sprintf("[::1]:%d", securePort), pki.CACert, kubeconfig.crt, kubeconfig.key); err != nil {
```

Comment:
shouldn't we use a conditional localhost here as well?

Reply:
yes, thanks for the catch (see Louise' comment)